comp/core/workloadmeta/types.go

```diff
@@ -984,6 +984,9 @@ type CollectorEvent struct {
 	Entity Entity
 }
 
+// PushEvent is an event that is pushed into metadata store by an external source
+type PushEvent CollectorEvent
```

Comment:
This type aliasing is used in order to keep collectors fully abstracted away from external components.

Reply:
I'm not sure about the value that this alias adds.

Reply:
replied below